src/components/DesignTheme/Blocks/RSS/View.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import RssBody from './RssBody';
+
+const View = ({ data, properties, block, path }) => {
+  return (
+    <div className={cx('block listing', data.template)}>
```

Comment:
Ocio che c'è scritto ancora `block listing`, c'è da sistemare la classe. Nell'edit ci viene messa in automatico come classe la chiave dell'oggetto config del sito (vedi commento sotto), potremmo mettere la stessa per comodità.